ui/app/components/children-status-bar.js

```diff
@@ -0,0 +1,27 @@
+import { computed } from '@ember/object';
+import DistributionBar from './distribution-bar';
+
+export default DistributionBar.extend({
```

Comment:
I definitely agree it would be clearer and more idiomatic. I just still have a gut aversion to making extra components. Especially when those components are in lists.

There are no lists in Nomad long enough to cause component allocation to drag the app though, so I shouldn't prematurely optimize.